homeassistant/components/mqtt/light/schema_basic.py

```diff
@@ -495,8 +495,10 @@ def _rgbx_received(
                 self._attr_color_mode = color_mode
             if self._topic[CONF_BRIGHTNESS_STATE_TOPIC] is None:
                 rgb = convert_color(*color)
-                percent_bright = float(color_util.color_RGB_to_hsv(*rgb)[2]) / 100.0
+                percent_bright = max(rgb) / 255.0
```

Comment:
Why do we divide by 255 here, only to multiply by 255 on the next line?
Let's also fix the lying variable name while we're at it:
```suggestion
                brightness = max(rgb)
```